examples/generative/corrdiff/datasets/dataset.py

```diff
@@ -55,13 +55,16 @@ def init_train_valid_datasets_from_config(
     """
 
     config = copy.deepcopy(dataset_cfg)
-    train_test_split = config.pop("train_test_split", True)
+    train_test_split = config.pop("train_test_split", False)
```

Comment:
I'm a little bit confused with this. I remember I had to set it to True in one of my PRs but I can't find it right now... I'll take a closer look tomorrow.

Reply:
This issue should be resolved in https://github.com/NVIDIA/modulus/pull/676/commits/dce69d02ca0e683040a5538a63bd33028bc4304a.

Reply:
The changes look good to me. Thanks!